src/dsp_tools/commands/ingest_xmlupload/bulk_ingest_client.py

```diff
@@ -70,3 +79,52 @@ def upload_file(
 
         logger.info(f"Uploaded file '{filepath}' to '{url}'")
         return None
+
+    def trigger_ingest_process(self) -> None:
+        """Start the ingest process on the server."""
+        url = f"{self.dsp_ingest_url}/projects/{self.shortcode}/bulk-ingest"
+        res = self.session.post(url, timeout=5)
+        if res.status_code in [STATUS_UNAUTHORIZED, STATUS_FORBIDDEN]:
+            raise BadCredentialsError("Unauthorized to start the ingest process. Please check your credentials.")
+        if res.status_code == STATUS_NOT_FOUND:
+            raise UserError(
+                f"No assets have been uploaded for project {self.shortcode}. "
+                "Before using the 'ingest-files' command, you must upload some files with the 'upload-files' command."
+            )
+        if res.status_code == STATUS_CONFLICT:
+            msg = f"Ingest process on the server {self.dsp_ingest_url} is already running. Wait until it completes..."
```

Comment:
The function returns exactly as in the case when a new bulk-ingest was started. Does it make sense to ask the user to `[w]ait until it completes`?

Reply:
> The function returns exactly as in the case when a new bulk-ingest was started. 

Yes, because it doesn't matter. As long as the mapping CSV is not ready, we have to wait.

> Does it make sense to ask the user to [w]ait until it completes?

I designed this command to wait until the mapping has been retrieved. It is documented that this might take very long: https://github.com/dasch-swiss/dsp-tools/blob/39898e83c7c4ac3ecb9541698bf6dd7cdf902ab1/docs/cli-commands.md#L271

In the meanwhile, the user sees a print output every 10 seconds: "Ingest process is still running. Wait until it completes..."